app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+import os
+
+
+class CommandError(Exception):
+    pass
+
+
+def move_file(command: str) -> None:
+    action, old_file, road_to_new_file = command.split()
+    if action != "mv":
+        raise CommandError('Command have to start with "mv"!')
+
+    start_dir = os.getcwd()
+
+    *to_file, new_file_name = road_to_new_file.split("/")
+    new_path = ""
+
+    for direct in to_file:
+        new_path = os.path.join(new_path, direct)
+        os.mkdir(new_path)
+    os.chdir(new_path)
```

Comment:
you can write it in one line 
```
os.makedirs(os.path.dirname(road_to_new_file))
```

Reply:
I don't quite understand how to do it

Reply:
<img width="819" alt="image" src="https://user-images.githubusercontent.com/93856423/218329290-1ea228de-0c08-411f-9ace-162a00dc1834.png">

1) when I test the code that you have now, it does not work

Reply:
you can replace lines 13-21 with this command 
```
os.makedirs(os.path.dirname(road_to_new_file))
```